structure/0055-mappings-team.md

```diff
@@ -0,0 +1,70 @@
+# Mappings team restructure
+
+## Summary
+
+The mappings team was originally created in [RFC 18][RFC18], but has since
+evolved: a triage team was created for [Quilt Mappings][QM], and the need to
+split the mappings team into smaller subteams has appeared. This RFC describes
+the proposed structure for the top-level team and those subteams.
+
+
+## Motivation
+
+The scope of the repositories maintained by the main mappings team is very big,
+from the main Quilt Mappings repository, to all of the toolchain used and 
+related to it. By splitting the mappings team, its members can be assigned to
+projects more specifically, and review requests in PRs would only reach to the
+repository maintainers rather than the whole team.
+
+
+## Explanation
+
+The mappings team would be split into three sub-teams: Quilt Mappings, Mappings
+Tooling, and the existing Mappings Triage sub-team.
+
+The Mappings Triage team is only in charge of reviewing mappings submitted to
+QM, ensuring their quality and making sure they stick to the conventions.
+This team only has triage access to the QM repository.
+
+The Quilt Mappings team would be in charge of maintaining QM, and also shares
+the responsibilities of the triage team. The main difference between the triage
+team and the QM team is their access level to the repository, meaning only QM
+members can merge PRs or push commits directly.
+
+The Mappings Tooling team would be in charge of maintaining all of the mappings
+toolchain (ie. [Mappings Hasher][Hasher], [Enigma], [Tiny Remapper][TR], etc.),
+and subsequently for reviewing PRs to these repositories.

```

Comment:
This section appears to have mixed tenses, which makes reading harder than it should be. Additionally, the terminology is inconsistent - is it named `Mappings Team` or `Quilt Mappings Team`?

I'd also avoid abbreviating the team name to `QM Team` to avoid confusion with the QM project, which has the same abbreviation.

Additionally, all team names should be capitalized - `Mappings Team`, not `mappings team`